tasks/audit-cat3.yml

```diff
@@ -1,6 +1,152 @@
-- name: "Place holder for Cat III Audits"
-  command: "true"
+---
+- name: "LOW | RHEL-07-010490 | PATCH | Unnecessary default system accounts must be removed."
+  shell: grep  {{ item }} /etc/passwd | wc -l
+  register: rhel_07_010490_audit
+  failed_when: no
+  changed_when: no
+  ignore_errors: yes
+  with_items:
+    - games
+    - ftp
+  tags:
+    - cat3
+    - low
+    - audit
+    - RHEL-07-010490
+
+- fail: msg="Unecessary default account present."
```

Comment:
But more importantly, we don't want to insert things that cause the role to fail. It needs to run to completion by design, and report via a template output at the end (currently unbuilt) the status of findings.

Failing immediately makes sense when running from the command line, but is a very poor experience when run from Tower, or if the job template is being run from other service via an API call to Tower.